docs/SETUP_CI_CD_PIPELINE.md

```diff
@@ -253,6 +253,22 @@ sd_gigadb:
 
 TODO: link up to @pli888's aws docs
 
+#### AWS dashboard
+
+There are two activities to perform on the AWS dashboard's EC2 console prior to using the GitLab Pipeline for deployment:
+
+1. Creation of Elastic IPs (under ``Network & Security > Elastic IPs``) to be used for the deployments to ``staging`` and ``live`` environments
```

Comment:
Just a note - it seems AWS only allows us a maximum of 5 EIPs. If 2 developers have EIPs for deploying to staging and live servers then the 3rd developer can only create one more EIP for deployment testing.

Reply:
And we don't even have the upstream deployment pipeline yet